Sources/Sentry/Profiling/SentryProfilingScreenFramesHelper.h

```diff
@@ -0,0 +1,17 @@
+#import "SentryDefines.h"
+#import <Foundation/Foundation.h>
+
+#if SENTRY_HAS_UIKIT
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class SentryScreenFrames;
+
+// Helper to use SentryScreenFrames without importing Swift on ObjC++ files.
```

Comment:
`m`: Please add more documentation or reference existing documentation what exactly this means and why it is necessary. Knowledge of contributors on swift-objc++ interop could be limited